sorc/ncep_post.fd/FILL_PSETFLD.f

```diff
@@ -1,39 +1,22 @@
 !> @file
-!                .      .    .     
-!> SUBPROGRAM:    READCNTRLgrb2_xml  READS POST xml CONTROL FILE
-!!   PRGRMMR: J. WANG         ORG: NCEP/EMC   DATE: 12-01-27       
-!!     
-!! ABSTRACT:
-!!     THIS ROUTINE SET THE OUTPUT FIELD GRIB2 INFORMATION SUCH    
-!!     AS PARAMETER NAME, LEVEL TYPE ETC FROM POST AVAILABLE FIELD
-!!     TABLE
-!!     
-!! PROGRAM HISTORY LOG:
-!!   01_27_2012  Jun Wang - INITIAL CODE
-!!   04_03_2012  Jun Wang - Add table info
-!!   03_10_2015  Lin Gan  - Using flat file data
-!!     
-!! USAGE:    CALL READCNTRL_XML(kth,kpv,pv)
-!!   INPUT ARGUMENT LIST:
-!!     param_ofld: output field
-!!     param_afld: available field in POST
-!!
-!!   OUTPUT ARGUMENT LIST: 
-!!     param_ofld: output field
-!!     
-!!   OUTPUT FILES:
-!!     NONE
-!!     
-!!   SUBPROGRAMS CALLED:
-!!     UTILITIES:
-!!
-!!     LIBRARY:
-!!       MODULE:   - xml_data_post_t
-!!     
-!!   ATTRIBUTES:
-!!     LANGUAGE: FORTRAN
-!!     MACHINE : IBM      
-!!
+!> @brief fill_psetfld reads post xml control file.
```

Comment:
When you use a function/subroutine name in the documentation, always lower-case it and follow with parens, like this:

`> @brief fill_psetfld() reads post xml control file.`

Reply:
Thanks for your comment. I will change them all.